pycds/climate_baseline_helpers.py

```diff
@@ -0,0 +1,169 @@
+from __future__ import print_function
+
+"""Tools for loading climate baseline data into database from flat files.
+"""
+
+import struct
+import datetime
+from calendar import monthrange
+from pycds import Network, History, Variable, DerivedValue
+
+pcic_climate_variable_network_name = 'PCIC Climate Variables'
+
+
+class InvalidVariableName(Exception):
+    pass
+
+
+def get_or_create_pcic_climate_variables_network(session):
+    """Get or, if it does not exist, create the synthetic network for derived variables
+
+    Args:
+        session (...): SQLAlchemy session for accessing the database
+
+    Returns:
+        Network object for synthetic network for derived variables
+    """
+
+    network = session.query(Network).filter(Network.name == pcic_climate_variable_network_name).first()
+    if not network:
+        network = Network(
+            name=pcic_climate_variable_network_name,
+            long_name='Synthetic network for climate variables computed by PCIC',
+            publish=True,
+            # color = '#??????', # TODO: Does this need to be defined?
+        )
+        session.add(network)
+        session.flush()
+    return network
+
+
+def create_pcic_climate_baseline_variables(session):
+    """Create, if they do not exist, the derived variables for climate baseline values.
+    Create the necessary synthetic network for them if it does not already exist.
+
+    Args:
+        session (...): SQLAlchemy session for accessing the database
+
+    Returns:
+        None
```

Comment:
While I realize that this is all side-effects based code (the functions are intended to do something in an external resource, the database), I still have this visceral reaction to functions that don't return anything.

Is there anything of value to return? A row count of created items? A boolean indicating success? Anything?